### PR TITLE
Update  /posts/handshakes_pagerank.md

### PR DESCRIPTION
**Pull request by bzz**
Very small drive-by fix: pickled graph filename and python snippet syntax

### CODE REVIEW

**Comment by bzz** [comment]
\cc @vmarkovtsev for reivew

**Comment by bzz** [comment]
Friendly _1 week_ ping @vmarkovtsev @mcuadros.

Does anybody know, who has a 🔑  from the blog?

File: content/post/handshakes_pagerank.md [hunk 1]
```diff
@@ -73,17 +73,17 @@ It is a pickled `scipy.sparse.csr_matrix`, the following Python code loads it:
...
-ndevs = graph.shape[0] - len(repos)
-print(repr(graph))
-print("Number of developers:", ndevs)
-print("Number of repositories:", len(repos))
-inv_repos = {r: i + ndevs for i, r in enumerate(repos)}
-name = "src-d/go-git"
-print("Repository %s has %d contributors, %d commits" % (
-    name, len(graph[inv_repos[name]].indices), graph[inv_repos[name]].sum()
-))
+    ndevs = graph.shape[0] - len(repos)
```
Comment (vmarkovtsev) [inline content/post/handshakes_pagerank.md]: This is Python: variables do not have a local scope (function closures are present though). It is a common practice to read and close the file as soon as possible, and continue with the loaded objects. So no need for this extra indentation.

**Comment by vmarkovtsev** [comment]
Wow Alex sorry, I completely missed this last week. I guess my brain was poisoned by the flu :)

Maximo and me have :key:

**Comment by bzz** [comment]
Got it, @vmarkovtsev thanks for review and feedback! History re-written

**Pull request merged**